homeassistant/components/binary_sensor/velbus.py

```diff
@@ -0,0 +1,82 @@
+"""
+Support for Velbus switches.
```

Comment:
This is the `binary_sensor` platform. We use the term `switch` in a different context.